dive_core/tests/available_gpu_time_test.cpp

```diff
@@ -0,0 +1,215 @@
+/*
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+#include "dive_core/available_gpu_time.h"
+
+#include <filesystem>
+
+#include "gtest/gtest.h"
+
+namespace Dive
+{
+namespace
+{
+std::filesystem::path fp = TEST_DATA_DIR;
+
+TEST(AvailableGpuTiming, LoadFromCsv_Successful)
+{
+    AvailableGpuTiming g;
+    EXPECT_FALSE(g.IsValid());
+    EXPECT_TRUE(g.LoadFromCsv(fp / "mock_gpu_time.csv"));
+    EXPECT_TRUE(g.IsValid());
+}
+
+TEST(AvailableMetrics, LoadFromCsv_MalformedFrameFail)
```

Comment:
is there a specific reason that you prefer to use two different names for the test suite: AvailableGpuTiming and AvailableMetrics?